src/lib.rs

```diff
@@ -1671,7 +1813,10 @@ impl<'a> Drop for RenderPass<'a> {
 }
 
 impl<'a> ComputePass<'a> {
-    /// Sets the active bind group for a given bind group index.
+    /// Sets the active bind group for a given bind group index. The bind group layout
```

Comment:
we should change that a bit. The active pipeline doesn't matter at *this* step, it only matters at draw() time.